ChangeLog.txt

```diff
@@ -7,6 +7,69 @@
    Entries may not always be in chronological/commit order.
    See license at the end of file. */
 
+2023-11-15 22:58 UTC+0100 Phil Krylov (phil a t krylov.eu)
+  * contrib/3rd/sqlite3/*
+    * Updated local sqlite3 from 3.8.2 to 3.44.0.
+
+2023-11-15 22:55 UTC+0100 Phil Krylov (phil a t krylov.eu)
+  * contrib/hbsqlit3/hbsqlit3.ch
+    * Updated SQLite constants to version 3.44.0.
+    % Reordered hbsqlit3.ch to simplify diffing from sqlite.h.
+  * contrib/hbsqlit3/core.c
+    + SQLITE3_CREATE_FUNCTION(): Added support for function flags.
+
+2023-11-15 22:50 UTC+0100 Phil Krylov (phil a t krylov.eu)
+  * contrib/hbsqlit3/core.c
+  * contrib/hbsqlit3/hbsqlit3.ch
+    % Minor updates to hbsqlit3, borrowed from @vszakats's fork.
+
+2023-11-15 22:48 UTC+0100 Phil Krylov (phil a t krylov.eu)
+  * contrib/hbsqlit3/core.c
+  * contrib/sddsqlt3/core.c
+    * use new sqlite3_prepare_v3() when built against
+      sqlite 3.20.0 or upper.
+  * contrib/hbsqlit3/core.c
+    + SQLITE3_PREPARE() add 3rd parameter <nPrepFlags>
+      Usable if build against sqlite 3.20.0 or upper, ignored otherwise
+  * contrib/hbsqlit3/hbsqlit3.ch
+    + add SQLITE3_PREPARE_PERSISTENT constant
+  ; commit is borrowed from vszakats/hb@b33de96c81ac790f390e506160a4d1d8c2b5f4fe
+  ; 2017-08-01 20:23 UTC Viktor Szakats (vszakats users.noreply.github.com)
```

Comment:
So why not using this header as the header of the entry?
This look confusing an reattribute this work to someone else.
I don't approve.

Reply:
Please keep the original header for merged commit entries
copied as-is. The commit entry itself is part of the work,
and the above reattributes that to someone else.

It's also confusing and looks as if this entry was written by
someone else. The "borrowed from" doesn't make this clear.

This is what I have in mind:
```
2017-08-01 20:23 UTC Viktor Szakats (vszakats users.noreply.github.com)
   * contrib/hbsqlit3/core.c
   * contrib/sddsqlt3/core.c
     * use new sqlite3_prepare_v3() when built against
       sqlite 3.20.0 or upper.
   * contrib/hbsqlit3/core.c
     + SQLITE3_PREPARE() add 3rd parameter <nPrepFlags>
       Usable if build against sqlite 3.20.0 or upper, ignored otherwise
   * contrib/hbsqlit3/hbsqlit3.ch
     + add SQLITE3_PREPARE_PERSISTENT constant
   ; commit is borrowed from vszakats/hb@b33de96c81ac790f390e506160a4d1d8c2b5f4fe
```

Your name will appear as the committer in the merged commit anyway,
so that is recorded as well.